### PR TITLE
Release/v1.1.0

### PR DESCRIPTION
**Pull request by vikramsinghvirdi**
Release notes : 

- Added remediation job support for Azure

> - azure_blob_remove_public_access
> - azure_network_security_group_close_port_22
> - azure_network_security_group_close_port_3389
> - azure_storage_account_allow_https_traffic_only
> - azure_vm_close_port_22

- Bug fixes

> - [s3_enable_logging_job keeps adding ACL permissions](https://github.com/vmware-samples/secure-state-remediation-jobs/issues/14)

### CODE REVIEW

**Review by mzkhan — DISMISSED** [review]

**Review by mzkhan — APPROVED** [review]
LGTM

**Review by vikramsinghvirdi — COMMENTED** [review]
lgtm

**Review by martinianodl — APPROVED** [review]

File: remediation_worker/jobs/azure_blob_remove_public_access/azure_blob_remove_public_access.py [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+import logging
+
+from azure.mgmt.storage import StorageManagementClient
+from azure.common.credentials import ServicePrincipalCredentials
+from azure.mgmt.storage.models import PublicAccess
+
+logging.basicConfig(level=logging.INFO)
+
+
+def logcall(f, *args, **kwargs):
```
Comment (regaw-leinad) [inline remediation_worker/jobs/azure_blob_remove_public_access/azure_blob_remove_public_access.py:27]: These `logcall` functions are never used. We should remove them if this is in fact the case.

**Review by bhanu13 — APPROVED** [review]

File: remediation_worker/jobs/azure_blob_remove_public_access/azure_blob_remove_public_access.py (hunk 1 above)
Comment (mzkhan) [reply remediation_worker/jobs/azure_blob_remove_public_access/azure_blob_remove_public_access.py:27]: We will take that removal in the next PR

**Pull request merged**